cypress/integration/metaSpec.js

```diff
@@ -0,0 +1,27 @@
+describe('The example page with imgix meta tags', () => {
+  it('successfully loads', () => {
+    cy.visit('/cypress/fixtures/samplePage.html');
+    // .should('equal', 'ix:useHttps');
+    cy.get('meta').each(($el) => {
+      const property = $el.attr('property');
+      if (!!property) {
+        expect(property).to.include('ix');
+      }
+    });
+  });
+  it('successfully applies imgix paramenters to the source url', () => {
+    cy.get('img').each(($el, index) => {
```

Comment:
If there are no `img`s on the page then this test succeeds when it should fail. This can be improved on by checking that the number of `img`s that are queried are the same as expected. The test above also suffers from a similar isssue if there are no meta tags.